tests/Tests.elm

```diff
@@ -116,6 +119,132 @@ test =
         ]
 
 
+expectEqualMultiline : String -> String -> Expect.Expectation
+expectEqualMultiline exp actual =
+    if exp == actual then
+        Expect.pass
+
+    else
+        let
+            context : number
+            context =
+                4
+
+            diff : List String
+            diff =
+                Diff.diffWith (\l r -> String.trim l == String.trim r)
+                    (String.lines exp)
+                    (String.lines actual)
+                    |> gatherGroups
+                    |> List.concatMap
+                        (\( head, tail ) ->
+                            case head of
+                                Diff.NoChange _ _ ->
+                                    if
+                                        (List.length tail > 2 * context)
+                                            && List.all (\t -> not (isChange t)) tail
+                                    then
+                                        head
+                                            :: List.take (context - 1) tail
+                                            ++ Diff.NoChange "" ""
+                                            :: Diff.NoChange "---" "---"
+                                            :: Diff.NoChange "" ""
+                                            :: List.reverse (List.take context (List.reverse tail))
+
+                                    else
+                                        head :: tail
+
+                                _ ->
+                                    head :: tail
+                        )
+                    |> List.map changeToString
+
+            header : String
+            header =
+                Ansi.Color.fontColor Ansi.Color.blue "Diff from expected to actual:"
+        in
+        Expect.fail (String.join "\n" (header :: diff))
+
+
+            changeToString : Diff.Change String -> String
+            changeToString change =
```

Comment:
Something weird happened here and in a few other places. I fixed it after the merge though